app/Services/PostService.php

```diff
@@ -65,7 +65,7 @@ public function update($signup, $data, $transactionId)
         // @TODO: This will is only temporary and will be removed!
         // If this is a signup update, get the most recent post.
         // If there is a quantity_pending, this is a signup.
-        if ($post->quantity_pending) {
+        if ($post->quantity || $post->quantity_pending) {
```

Comment:
I'm confused about what this whole conditional is used for. It seems like `$post = $this->repository->update($signup, $data);` above would take care of everything we'd need?